src/index.tsx

```diff
@@ -177,34 +185,44 @@ export default class OtpInputs extends PureComponent<Props, State> {
       selectTextOnFocus,
       testIDPrefix,
       unfocusedBorderColor,
+      forceLTR,
+      isRTL,
+      debug,
+      placeholder,
     } = this.props
     const { otpCode } = this.state
     const iterationArray = Array<number>(numberOfInputs).fill(0)
 
-    return iterationArray.map((_, index) => (
-      <OtpInput
-        autoCapitalize={autoCapitalize}
-        clearTextOnFocus={clearTextOnFocus}
-        containerStyles={inputContainerStyles}
-        focusStyles={focusStyles}
-        error={!!errorMessage}
-        focusedBorderColor={focusedBorderColor}
-        handleBackspace={(event: TextInputOnKeyPressEventData) =>
-          this._handleBackspace(event, index)
-        }
-        inputStyles={inputStyles}
-        key={index}
-        secureTextEntry={secureTextEntry}
-        keyboardType={keyboardType}
-        ref={this.inputs[index]}
-        selectTextOnFocus={selectTextOnFocus}
-        textErrorColor={inputTextErrorColor}
-        unfocusedBorderColor={unfocusedBorderColor}
-        updateText={(event: TextInputOnChangeEventData) => this._updateText(event, index)}
-        value={otpCode[index]}
-        testID={`${testIDPrefix}-${index}`}
-      />
-    ))
+    return iterationArray.map((_, index) => {
+      if(forceLTR && isRTL){
```

Comment:
Do wee need 2 props which should do the same thing? I don't see actual usage of this differently.

Reply:
we "can" have just one, but in case someone wants to let them enter in the reverse order it will be helpful. There is also the case where someone maybe using this in the current behaviour, and a minor update would break their code. I have seen people using the code where they manually reverse the OTP if RTL is enabled.

Reply:
No, LTR is just by default. We shoudn't introduce 2 new props for the same, as you do not use them separately :)